### DISCUSSION TITLE
Welcome to trice Discussions!

### DISCUSSION THREAD

**Discussion by rokath** [rokath/trice#98] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rokath** [comment]
TO SEE **ALL** DISCUSSIONS CLICK `Filter->All`.

**Comment by Rafi22222** [comment]
Hi ,
I'm Rafi. Most of the time I develop SW related to HW and communication.
I hope get some knowledge from the community and contribute (if I get to something
meaningful).
